### PR TITLE
Forms/Deploy release notes

### PR DESCRIPTION
**Pull request by AndyButland**
This PR migrates the last couple of years worth of release notes into markdown and as pages for the new docs site.

We're currently using our.umbraco.com for this information, but as that's less used now it's not really he best place - and this follows the location that Cloud are using.

I don't think you need to really review the detail as it's just copy/pasted (and converted) from the HTML on our; likely just a check over that the links and layout looks OK.

When this is merged I'll update the copy on our to reference these new pages for anyone looking there still or in future.

// @hemraker @BoletteKern

### CODE REVIEW

File: 10/umbraco-forms/release-notes.md [hunk 1]
```diff
@@ -0,0 +1,523 @@
...
+*   Ensured field references in copied forms are updated to the new fields [#815](https://github.com/umbraco/Umbraco.Forms.Issues/issues/815)
+*   Fixed validation of mandatory date field [#817](https://github.com/umbraco/Umbraco.Forms.Issues/issues/817)
+*   Added tag option for text/description field type [#821](https://github.com/umbraco/Umbraco.Forms.Issues/issues/821)
+*   Added additional input type options to text field type [#825](https://github.com/umbraco/Umbraco.Forms.Issues/issues/825)
+*   Restored open/edit options to form picker preview [#832](https://github.com/umbraco/Umbraco.Forms.Issues/issues/827)
+*   Improved performance of permission related queries [#832](https://github.com/umbraco/Umbraco.Forms.Issues/issues/827) (raised in discussion)
+
+### [10.0.5](https://github.com/umbraco/Umbraco.Forms.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F10.0.5) (July 14th 2022)
+
+*   Fixed macro partial view tree so RCL shipped partials from Forms are only shown in the "picker" dialog. [#814](https://github.com/umbraco/Umbraco.Forms.Issues/issues/814)
```
Comment (eshanrnh) [inline 10/umbraco-forms/release-notes.md:163]: Can we define RCL here? Is it Razor Class Library 🤔?
```suggestion
*   Fixed macro partial view tree so Razor Class Library (RCL) shipped partials from Forms are only shown in the "picker" dialog. [#814](https://github.com/umbraco/Umbraco.Forms.Issues/issues/814)
```

File: 10/umbraco-forms/release-notes.md [hunk 2]
```diff
@@ -0,0 +1,523 @@
...
+*   Added workflow audit trail.
+*   Added workflow retry option.
+*   Added option to customize behaviour of default workflows, including mandatory workflows [#654](https://github.com/umbraco/Umbraco.Forms.Issues/issues/654)
+*   Added conditional workflows [#370](https://github.com/umbraco/Umbraco.Forms.Issues/issues/370)
+*   Added ability to redirect to an external site from workflows, after all have completed.
+*   Extended the form picker to use folder structure [#729](https://github.com/umbraco/Umbraco.Forms.Issues/issues/729​)
+*   Added option for prevalue captions [#84](https://github.com/umbraco/Umbraco.Forms.Issues/issues/84​)
+*   Ensured user group start folder aggregation for user's permissions doesn't include user groups that don't have access to Forms [#772](https://github.com/umbraco/Umbraco.Forms.Issues/issues/772)
+*   Added option for creating permissions on form for user groups to all groups, or all groups the creating user is part of.
+*   Provided fixes for issues with rendering localized dates in the back-office entries view [#777](https://github.com/umbraco/Umbraco.Forms.Issues/issues/777)
```
Comment (eshanrnh) [inline 10/umbraco-forms/release-notes.md:178]:
```suggestion
*   Provided fixes for issues with rendering localized dates in the backoffice entries view [#777](https://github.com/umbraco/Umbraco.Forms.Issues/issues/777)
```

File: 10/umbraco-forms/release-notes.md [hunk 3]
```diff
@@ -0,0 +1,523 @@
...
+*   Added read-only, rich text data type (V9 and 10).
```
Comment (eshanrnh) [inline 10/umbraco-forms/release-notes.md:179]:
```suggestion
*   Added read-only, rich text Data Type (V9 and 10).
```

File: 10/umbraco-forms/release-notes.md [hunk 4]
```diff
@@ -0,0 +1,523 @@
...
+*   Prevented the previous button on multi-page forms from triggering validation (which involved an update to the client-side validation library we have a dependency on, when the website is not referencing jQuery) [#741](https://github.com/umbraco/Umbraco.Forms.Issues/issues/741)​
+*   Removed reliance on class names for multi-page form navigation to allow removal in custom themes [#740](https://github.com/umbraco/Umbraco.Forms.Issues/issues/740)
+*   Added details of file upload supported extensions to the view model (that can be used in custom themes or field types) [#744](https://github.com/umbraco/Umbraco.Forms.Issues/issues/744)
+*   Remove inline scripts from the reCAPTCHA field type (completing the removal of all inline scripts started in the previous release and allowing for the setting of a stricter content security policy) [#745](https://github.com/umbraco/Umbraco.Forms.Issues/issues/745)
+*   Fixed issue with date display in the back-office when localized date formats are in use [#747](https://github.com/umbraco/Umbraco.Forms.Issues/issues/747)
```
Comment (eshanrnh) [inline 10/umbraco-forms/release-notes.md:242]:
```suggestion
*   Fixed issue with date display in the backoffice when localized date formats are in use [#747](https://github.com/umbraco/Umbraco.Forms.Issues/issues/747)
```

File: 10/umbraco-forms/release-notes.md [hunk 5]
```diff
@@ -0,0 +1,523 @@
...
+*   Fixed issue with use of back button returning to form and displaying submission message when previously having redirected to a new page.
+*   Fixed issue with access to forms in folders for users with a single start folder defined. [#759](https://github.com/umbraco/Umbraco.Forms.Issues/issues/759)
+*   Fixed issue with access to previously created forms for non-admin users. [#764](https://github.com/umbraco/Umbraco.Forms.Issues/issues/764)
+*   Resolves issue where an authenticated user with access to Forms can enumerate permissions related to forms access for other users.
+
+### [8.11.0](https://github.com/umbraco/Umbraco.Forms.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F8.11.0) and [9.3.0](https://github.com/umbraco/Umbraco.Forms.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F9.3.0) (March 8th 2022)
+
+*   All changes listed under 8.11.0-rc001 and 9.3.0-rc001
+*   Fixed issue with clean on already cleaned project (V9 only) [#732](https://github.com/umbraco/Umbraco.Forms.Issues/issues/732)
+*   Resolved client-side error when just the jquery unobtrusive dependency is missing [#734](https://github.com/umbraco/Umbraco.Forms.Issues/issues/734)
```
Comment (eshanrnh) [inline 10/umbraco-forms/release-notes.md:260]:
```suggestion
*   Resolved client-side error when the jquery unobtrusive dependency is missing [#734](https://github.com/umbraco/Umbraco.Forms.Issues/issues/734)
```

File: 10/umbraco-forms/release-notes.md [hunk 6]
```diff
@@ -0,0 +1,523 @@
...
+*   Fixed two typos in labels [#727](https://github.com/umbraco/Umbraco.Forms.Issues/issues/727)
+
+### [8.11.0-rc001](https://github.com/umbraco/Umbraco.Forms.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F8.11.0) and [9.3.0-rc001](https://github.com/umbraco/Umbraco.Forms.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F9.3.0) (February 15th 2022)
+
+*   Config for control over user access to new forms [#12](https://github.com/umbraco/Umbraco.Forms.Issues/issues/12)
+*   Management of form permissions by user group [#19](https://github.com/umbraco/Umbraco.Forms.Issues/issues/19)
+*   Separated permissions for form "design" and "entry viewer" [#3](https://github.com/umbraco/Umbraco.Forms.Issues/issues/3)
+*   Setting of start folders for users
+*   Added permission and feature for editing entries via the back-office [#498](https://github.com/umbraco/Umbraco.Forms.Issues/issues/498)
```
Comment (eshanrnh) [inline 10/umbraco-forms/release-notes.md:269]:
```suggestion
*   Added permission and feature for editing entries via the backoffice [#498](https://github.com/umbraco/Umbraco.Forms.Issues/issues/498)
```

File: 10/umbraco-forms/release-notes.md [hunk 7]
```diff
@@ -0,0 +1,523 @@
...
+*   Added migration and healthcheck for missing index following V7 upgrade [#713](https://github.com/umbraco/Umbraco.Forms.Issues/issues/713)
+*   Allow tracking of calculated score in reCAPTCHA checks [#664](https://github.com/umbraco/Umbraco.Forms.Issues/issues/664)
+*   Removed use of inline scripts allowing setting of a stricter content security policy [#677](https://github.com/umbraco/Umbraco.Forms.Issues/issues/677)
+*   Fixed typos in setting description (V9 only) [#710](https://github.com/umbraco/Umbraco.Forms.Issues/issues/710)
+*   Fixed timezone conversion on entries viewer [#723](https://github.com/umbraco/Umbraco.Forms.Issues/issues/723)
+*   Fixed null reference in back-office user check for retrieving records outside of a back-office request [#724](https://github.com/umbraco/Umbraco.Forms.Issues/issues/724)
```
Comment (eshanrnh) [inline 10/umbraco-forms/release-notes.md:275]:
```suggestion
*   Fixed null reference in backoffice user check for retrieving records outside of a backoffice request [#724](https://github.com/umbraco/Umbraco.Forms.Issues/issues/724)
```

File: 10/umbraco-forms/release-notes.md [hunk 8]
```diff
@@ -0,0 +1,523 @@
...
+*   Fixed case insensitive view name under forms security (V9 only) [#725](https://github.com/umbraco/Umbraco.Forms.Issues/issues/725)
+*   Additional translations for localized back-office: Czech and Danish.
```
Comment (eshanrnh) [inline 10/umbraco-forms/release-notes.md:277]:
```suggestion
*   Additional translations for localized backoffice: Czech and Danish.
```

File: 10/umbraco-forms/release-notes.md [hunk 9]
```diff
@@ -0,0 +1,523 @@
...
+
+*   Specified serialization settings used by Forms avoiding issues with changes to global defaults [#264](https://github.com/umbraco/Umbraco.Forms.Issues/issues/264)
+*   Resolved various issues related to conditional form logic [#623](https://github.com/umbraco/Umbraco.Forms.Issues/issues/623) [#686](https://github.com/umbraco/Umbraco.Forms.Issues/issues/686) [#689](https://github.com/umbraco/Umbraco.Forms.Issues/issues/689) [#693](https://github.com/umbraco/Umbraco.Forms.Issues/issues/693) [#695](https://github.com/umbraco/Umbraco.Forms.Issues/issues/695)
+*   Updated email template to support multiple file upload fields [#691](https://github.com/umbraco/Umbraco.Forms.Issues/issues/691)
+*   Fixed issue introduced by localization of workflow details [#692](https://github.com/umbraco/Umbraco.Forms.Issues/issues/692)
+*   Ensured list of licensed domains on dashboard includes the full set allocated to the license [#697](https://github.com/umbraco/Umbraco.Forms.Issues/issues/697)
+*   Fixed rendering of HTML entities in form titles in the back-office [#699](https://github.com/umbraco/Umbraco.Forms.Issues/issues/699)
```
Comment (eshanrnh) [inline 10/umbraco-forms/release-notes.md:301]:
```suggestion
*   Fixed rendering of HTML entities in form titles in the backoffice [#699](https://github.com/umbraco/Umbraco.Forms.Issues/issues/699)
```

File: 10/umbraco-forms/release-notes.md [hunk 10]
```diff
@@ -0,0 +1,523 @@
...
+*   Fixed issue with back-office delete of form with > 2000 associated entries [#700](https://github.com/umbraco/Umbraco.Forms.Issues/issues/700)
```
Comment (eshanrnh) [inline 10/umbraco-forms/release-notes.md:302]:
```suggestion
*   Fixed issue with backoffice delete of form with > 2000 associated entries [#700](https://github.com/umbraco/Umbraco.Forms.Issues/issues/700)
```

File: 10/umbraco-forms/release-notes.md [hunk 11]
```diff
@@ -0,0 +1,523 @@
...
+*   Additional settings for text fields
+*   Enhanced security for file uploads [#11](https://github.com/umbraco/Umbraco.Forms.Issues/issues/11)
+*   Fixed issues relating to the Umbraco Documents prevalue source [#638](https://github.com/umbraco/Umbraco.Forms.Issues/issues/638)
+*   Added details available in save events to detect and act on forms or folders being moved [#667](https://github.com/umbraco/Umbraco.Forms.Issues/issues/667)
+*   Applied dictionary translations to form fields displayed in backoffice entries viewer. [#672](https://github.com/umbraco/Umbraco.Forms.Issues/issues/672)
+*   Resolved issue with field type script rendering when multiple forms are displayed on a page. [#670](https://github.com/umbraco/Umbraco.Forms.Issues/issues/670)
+*   Completed support for client-side views to be created outside of the _App\_Plugins_ folder, thus being retained following a _dotnet clean_ (V9 only). [#13](https://github.com/umbraco/Umbraco.Forms.Issues/issues/13)
+*   Added fallback to default configured SMTP sender address (V9 only). [#676](https://github.com/umbraco/Umbraco.Forms.Issues/issues/676)
```
Comment (eshanrnh) [inline 10/umbraco-forms/release-notes.md:323]:
```suggestion
*   Added fallback to default configured Simple Mail Transfer Protocol (SMTP) sender address (V9 only). [#676](https://github.com/umbraco/Umbraco.Forms.Issues/issues/676)
```

File: 10/umbraco-forms/release-notes.md [hunk 12]
```diff
@@ -0,0 +1,523 @@
...
+
+*   Conditional display of "submit" or "next/previous" buttons [#18](https://github.com/umbraco/Umbraco.Forms.Issues/issues/18)
+*   Include form details in "Sent to URL" workflow [#569](https://github.com/umbraco/Umbraco.Forms.Issues/issues/569)
+*   Download submitted files organised by entry [#626](https://github.com/umbraco/Umbraco.Forms.Issues/issues/626)
+*   Ensured first field with validation error gets focus [#602](https://github.com/umbraco/Umbraco.Forms.Issues/issues/602)
+*   [](https://github.com/umbraco/Umbraco.Forms.Issues/issues/18)Fixed null reference exception when deleting records within workflows. [#100](https://github.com/umbraco/Umbraco.Forms.Issues/issues/100)
+*   Fixed issue where check-boxes are used in conditions. [#192](https://github.com/umbraco/Umbraco.Forms.Issues/issues/192)
+*   Added missing custom CMS class to text field template. [#484](https://github.com/umbraco/Umbraco.Forms.Issues/issues/484)
+*   Show list of licensed domains in the back-office. [#629](https://github.com/umbraco/Umbraco.Forms.Issues/issues/629)
```
Comment (eshanrnh) [inline 10/umbraco-forms/release-notes.md:345]:
```suggestion
*   Show list of licensed domains in the backoffice. [#629](https://github.com/umbraco/Umbraco.Forms.Issues/issues/629)
```

File: 10/umbraco-forms/release-notes.md [hunk 13]
```diff
@@ -0,0 +1,523 @@
...
+*   Restored behaviour of excluding scripts when rendering forms to only do so when explicitly requested to. [#634](https://github.com/umbraco/Umbraco.Forms.Issues/issues/634)
+*   Added a new event to support hooking into form entry display in the back-office. [#639](https://github.com/umbraco/Umbraco.Forms.Issues/issues/639)
```
Comment (eshanrnh) [inline 10/umbraco-forms/release-notes.md:347]:
```suggestion
*   Added a new event to support hooking into form entry display in the backoffice. [#639](https://github.com/umbraco/Umbraco.Forms.Issues/issues/639)
```

File: 10/umbraco-forms/release-notes.md [hunk 14]
```diff
@@ -0,0 +1,523 @@
...
+*   Ensured when forms are created from templates that they have unique page, fieldset and field Ids. [#647](https://github.com/umbraco/Umbraco.Forms.Issues/issues/647)
+*   Fixed issue with saving of forms with sensitive data by editors not in the sensitive data user group. [#652](https://github.com/umbraco/Umbraco.Forms.Issues/issues/652)
+*   Removed display of fields in email template that have no expected user input. [#659](https://github.com/umbraco/Umbraco.Forms.Issues/issues/659)
+*   Amended the “-ing” events (e.g. “Saving”) to be cancellable and allow changes to the object being saved. [#663](https://github.com/umbraco/Umbraco.Forms.Issues/issues/663)
+*   Added a field type property to hide the mandatory option where it’s not appropriate (i.e. where there’s no expected user input). [#665](https://github.com/umbraco/Umbraco.Forms.Issues/issues/665)
```
Comment (eshanrnh) [inline 10/umbraco-forms/release-notes.md:352]:
```suggestion
*   Added a field-type property to hide the mandatory option where it’s not appropriate (that is where there’s no expected user input). [#665](https://github.com/umbraco/Umbraco.Forms.Issues/issues/665)
```

File: 10/umbraco-forms/release-notes.md [hunk 15]
```diff
@@ -0,0 +1,523 @@
...
+*   Autocomplete attributes for forms and fields [#322](https://github.com/umbraco/Umbraco.Forms.Issues/issues/322)
+*   Workflow UX tidy-up [#334](https://github.com/umbraco/Umbraco.Forms.Issues/issues/334)
+*   Fixed prevalue sources not finding grandchildren [#365](https://github.com/umbraco/Umbraco.Forms.Issues/issues/365)
+*   Configuration of a default theme [#398](https://github.com/umbraco/Umbraco.Forms.Issues/issues/398)
+*   Added busy indicator when exporting to Excel and performing record set operations [#419](https://github.com/umbraco/Umbraco.Forms.Issues/issues/419) [#575](https://github.com/umbraco/Umbraco.Forms.Issues/issues/575)
+*   Ensured form submissions with message display follow post/redirect/get pattern [#485](https://github.com/umbraco/Umbraco.Forms.Issues/issues/485) [#572](https://github.com/umbraco/Umbraco.Forms.Issues/issues/572) [#593](https://github.com/umbraco/Umbraco.Forms.Issues/issues/593)
+*   Added ability to toggle field labels [#530](https://github.com/umbraco/Umbraco.Forms.Issues/issues/530)
+*   Added link to page where form was submitted from back-office entry screen [#607](https://github.com/umbraco/Umbraco.Forms.Issues/issues/607)
```
Comment (eshanrnh) [inline 10/umbraco-forms/release-notes.md:384]:
```suggestion
*   Added link to a page where the form was submitted from backoffice entry screen [#607](https://github.com/umbraco/Umbraco.Forms.Issues/issues/607)
```

File: 10/umbraco-forms/release-notes.md [hunk 16]
```diff
@@ -0,0 +1,523 @@
...
+
+*   Resolution of security vulnerability (see [blog post](https://umbraco.com/blog/security-advisory-20th-of-july-2021-patch-is-now-available/)).
+
+### [8.7.5](https://github.com/umbraco/Umbraco.Forms.Issues/issues?q=label%3Arelease%2F8.7.5+is%3Aclosed) (July 6th 2021)
+
+*   Resolved JavaScript incompatibility issues with IE11 [#601](https://github.com/umbraco/Umbraco.Forms.Issues/issues/601)
+*   Fixed bug with sending static values in "Send to URL" workflow [#597](https://github.com/umbraco/Umbraco.Forms.Issues/issues/597)
+*   Displayed visual indicator of conditions applied to form groups] [#590](https://github.com/umbraco/Umbraco.Forms.Issues/issues/590)
+*   Handled escaping of pre-values with apostrophes when used in conditions [#456](https://github.com/umbraco/Umbraco.Forms.Issues/issues/456)
+    *   _Note that this change has required a minor amend to the Script.cshtml partial view, so if you've modified this in your installation and don't simply copy over from the update, you should review to ensure you apply the update._
```
Comment (eshanrnh) [inline 10/umbraco-forms/release-notes.md:405]:
```suggestion
    *   _**Note:** This change has required a minor amendment to the Script.cshtml partial view. So if you've modified this in your installation, don't copy over from the update. You should review it to ensure you apply the update._
```

File: 10/umbraco-forms/release-notes.md [hunk 17]
```diff
@@ -0,0 +1,523 @@
...
+
+*   Resolved exception thrown when editing a form entry via an invalid ID or with an ID for a different form [#584](https://github.com/umbraco/Umbraco.Forms.Issues/issues/584)
+*   Fixed issue where editing a form entry with a deleted field generates an exception [#583](https://github.com/umbraco/Umbraco.Forms.Issues/issues/583)
+*   Fixed issue with non-active workflows being processed [#582](https://github.com/umbraco/Umbraco.Forms.Issues/issues/582)
+*   Fixed error triggered by form containing no fields that store data [#580](https://github.com/umbraco/Umbraco.Forms.Issues/issues/580)
+*   Reduced level of log messages related to reCAPTCHA field type [#573](https://github.com/umbraco/Umbraco.Forms.Issues/issues/573)
+*   Resolved issue with use of reCAPTCHA field type on multiple forms on the same page [#571](https://github.com/umbraco/Umbraco.Forms.Issues/issues/571)
+*   Resolved issue with copy form function when form storage configuration key is missing [#567](https://github.com/umbraco/Umbraco.Forms.Issues/issues/567)
+*   Resolved bug with back-office data filter in non-English locales - [#560](https://github.com/umbraco/Umbraco.Forms.Issues/issues/560)
```
Comment (eshanrnh) [inline 10/umbraco-forms/release-notes.md:419]:
```suggestion
*   Resolved bug with backoffice data filter in non-English locales - [#560](https://github.com/umbraco/Umbraco.Forms.Issues/issues/560)
```

File: 10/umbraco-forms/release-notes.md [hunk 18]
```diff
@@ -0,0 +1,523 @@
...
+*   Removed use of obsolete methods in reCAPTCHA field type [#557](https://github.com/umbraco/Umbraco.Forms.Issues/issues/557)
+*   Added cache invalidation for member properties when used for "magic strings" [#534](https://github.com/umbraco/Umbraco.Forms.Issues/issues/534)
+
+### [8.7.3](https://github.com/umbraco/Umbraco.Forms.Issues/issues?q=label%3Arelease%2F8.7.3+is%3Aclosed) (May 18th 2021)
+
+*   Fixed issue with migrations using SQL CE [#559](https://github.com/umbraco/Umbraco.Forms.Issues/issues/559)
+*   Aligned text field maxlength attribute with database field size [#563](https://github.com/umbraco/Umbraco.Forms.Issues/issues/563)
+*   Removed unnecessary rendering of validation framework requirement when form previewed in a rich text editor [#562](https://github.com/umbraco/Umbraco.Forms.Issues/issues/562)
+*   Optimised loading of form record counts in the back-office dashboard [#561](https://github.com/umbraco/Umbraco.Forms.Issues/issues/561)
```
Comment (eshanrnh) [inline 10/umbraco-forms/release-notes.md:428]:
```suggestion
*   Optimised loading of form record counts in the backoffice dashboard [#561](https://github.com/umbraco/Umbraco.Forms.Issues/issues/561)
```

File: 10/umbraco-forms/release-notes.md [hunk 19]
```diff
@@ -0,0 +1,523 @@
...
+*   Added support for multiple forms on one page using recaptchaV3 [#556](https://github.com/umbraco/Umbraco.Forms.Issues/issues/556)
+*   Resolved JavaScript console errors found on back-office security settings page [#505](https://github.com/umbraco/Umbraco.Forms.Issues/issues/505)
```
Comment (eshanrnh) [inline 10/umbraco-forms/release-notes.md:430]:
```suggestion
*   Resolved JavaScript console errors found on the backoffice security settings page [#505](https://github.com/umbraco/Umbraco.Forms.Issues/issues/505)
```

File: 10/umbraco-forms/release-notes.md [hunk 20]
```diff
@@ -0,0 +1,523 @@
...
+*   Restored create datasource based on webservice functionality [#265](https://github.com/umbraco/Umbraco.Forms.Issues/issues/265)
+*   Resolved issue with save of uploaded files when a pre-populate event is registered [#177](https://github.com/umbraco/Umbraco.Forms.Issues/issues/177)
+
+### [8.7.2](https://github.com/umbraco/Umbraco.Forms.Issues/issues?q=label%3Arelease%2F8.7.2+is%3Aclosed) (May 11th 2021)
+
+*   Style back-office preview for Recaptcha3 field type [#552](https://github.com/umbraco/Umbraco.Forms.Issues/issues/552)
```
Comment (eshanrnh) [inline 10/umbraco-forms/release-notes.md:436]:
```suggestion
*   Style backoffice preview for Recaptcha3 field type [#552](https://github.com/umbraco/Umbraco.Forms.Issues/issues/552)
```

File: 10/umbraco-forms/release-notes.md [hunk 21]
```diff
@@ -0,0 +1,523 @@
...
+*   Fixed issue with Examine component registration [#535](https://github.com/umbraco/Umbraco.Forms.Issues/issues/535)
+*   Aligned styling of delete button with core [#528](https://github.com/umbraco/Umbraco.Forms.Issues/issues/528)
+*   Updated styling of copy form dialog [#527](https://github.com/umbraco/Umbraco.Forms.Issues/issues/527)
+*   Updated styling of disabled add column button for form groups [#526](https://github.com/umbraco/Umbraco.Forms.Issues/issues/526)
+*   Added option for creating a multi-select select list [#473](https://github.com/umbraco/Umbraco.Forms.Issues/issues/473)
+*   Added missing default form setting configuration entry [#369](https://github.com/umbraco/Umbraco.Forms.Issues/issues/369)
+*   Made the body text component of the "title and description" field type a text area [#326](https://github.com/umbraco/Umbraco.Forms.Issues/issues/326)
+*   Fixed issue with conditional field rendering [#207](https://github.com/umbraco/Umbraco.Forms.Issues/issues/207)
+*   Removed some legacy, commented out code from back-office JavaScript [#148](https://github.com/umbraco/Umbraco.Forms.Issues/issues/148)
```
Comment (eshanrnh) [inline 10/umbraco-forms/release-notes.md:450]:
```suggestion
*   Removed some legacy, commented-out code from backoffice JavaScript [#148](https://github.com/umbraco/Umbraco.Forms.Issues/issues/148)
```

File: 10/umbraco-forms/release-notes.md [hunk 22]
```diff
@@ -0,0 +1,523 @@
...
+*   Aligned caret in conditions overlay [#510](https://github.com/umbraco/Umbraco.Forms.Issues/issues/510)
+*   Removed use of turquoise colour [#511](https://github.com/umbraco/Umbraco.Forms.Issues/issues/511)
+*   Added clear line between conditions [#512](https://github.com/umbraco/Umbraco.Forms.Issues/issues/512)
+*   Prevented addition of empty file extension on file upload [#516](https://github.com/umbraco/Umbraco.Forms.Issues/issues/516)
+*   Improved UI for file upload types [#517](https://github.com/umbraco/Umbraco.Forms.Issues/issues/517)
+*   Prevented password managers acting on form group and page name fields [#518](https://github.com/umbraco/Umbraco.Forms.Issues/issues/518)
+*   Updated styling on re-order button [#519](https://github.com/umbraco/Umbraco.Forms.Issues/issues/519)
+*   Set upper limit on number of columns in form group [#520](https://github.com/umbraco/Umbraco.Forms.Issues/issues/520)
+*   Updated reCAPTCHA v3 to function on submut buttons as well as inputs [#521](https://github.com/umbraco/Umbraco.Forms.Issues/issues/521)
+*   Sanitise for field names when sort option provided in back-office record filter request.
```
Comment (eshanrnh) [inline 10/umbraco-forms/release-notes.md:472]:
```suggestion
*   Sanitise for field names when the sort option is provided in the backoffice record filter request.
```

**Review by eshanrnh — COMMENTED** [review]
Looks good to me, @AndyButland 🙌 I've reviewed only the Forms version 10 article and added comments (mostly Vale errors) to it. Please feel free to apply them for version 11 article as well.

File: 10/umbraco-deploy/release-notes.md [hunk 23]
```diff
@@ -0,0 +1,283 @@
...
+*   Fixed issue with transfer of nested content string values using item templates (i.e. commencing with "{{") that were erroneously considered as JSON.
+
+### [Deploy 4.7.0-rc001](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F4.7.0), [9.5.0-rc001](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F9.5.0) and [10.1.0-rc001](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F10.1.0) (September 7th 2022)
+
+*   Introduced and used caching in deploy operations to improve performance.
+*   Increased default and added setting for disk operation timeouts [#135](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/135)
+*   Single language content transfers [#132](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/132)
+*   Scheduled content transfers.
+*   Corrected transfer of unpublished content status [#131](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/131)
+*   Improved UX and descriptions in back-office settings dashboard [#118](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/118)
```
Comment (jonat123) [inline 10/umbraco-deploy/release-notes.md:68]:
```suggestion
*   Improved UX and descriptions in backoffice settings dashboard [#118](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/118)
```

File: 10/umbraco-deploy/release-notes.md [hunk 24]
```diff
@@ -0,0 +1,283 @@
...
+*   Added ability to download Deploy artifacts (.uda files) as a zip archive from the management dashboard.
+*   Added sort options to the schema comparison view in the management dashboard [#115](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/115)
+*   Indented the JSON representation of data type configuration details in the .uda files for ease of review [#85](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/85)
+*   Fixed issue with transfer of Forms prevalue sources from text files that include captions.
+*   Ensured document type validation messages are transferred between environments [#137](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/137)
```
Comment (jonat123) [inline 10/umbraco-deploy/release-notes.md:73]:
```suggestion
*   Ensured Document Type validation messages are transferred between environments [#137](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/137)
```

File: 10/umbraco-deploy/release-notes.md [hunk 25]
```diff
@@ -0,0 +1,283 @@
...
+
+### Deploy Contrib 4.2.0-rc001, 9.1.0-rc001 and 10.1.0-rc001 (September 7th 2022)
+
+*   Used caching in value connectors to improve performance.
+
+### [Deploy 4.6.2](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F4.6.2), [9.4.2](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F9.4.2) and [10.0.3](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F10.0.3) (August 16th 2022)
+
+*   Aligned Git URL displayed in back-office with that in Cloud Portal (V4 only) [#136](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/136)
```
Comment (jonat123) [inline 10/umbraco-deploy/release-notes.md:81]:
```suggestion
*   Aligned Git URL displayed in backoffice with that in Cloud Portal (V4 only) [#136](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/136)
```

File: 10/umbraco-deploy/release-notes.md [hunk 26]
```diff
@@ -0,0 +1,283 @@
...
+*   Fixed incorrect availability of workspace restore in production environment (V9 and V10 only)
+*   Added close button to "Transfer now" dialog
+*   Resolved registration of deployable types to support configuration for "back-office edition".
+
+### [Deploy 4.6.1,](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F4.6.1) [9.4.1](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F9.4.1) and [10.0.2](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F10.0.2) (July 12th 2022)
+
+*   Resolved issue with media restore when database items exist and files don't [#123](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/123)
+*   Added details of failed deployment to deploy dashboard [#120](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/120)
+*   Added copy button for deploy log in the back-office [#121](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/121)
```
Comment (jonat123) [inline 10/umbraco-deploy/release-notes.md:91]:
```suggestion
*   Added copy button for deploy log in the backoffice [#121](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/121)
```

File: 10/umbraco-deploy/release-notes.md [hunk 27]
```diff
@@ -0,0 +1,283 @@
...
+*   Fixed issue with deploying Form prevalues with Forms versions < 8.5 [#23](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/23)[](https://github.com/umbraco/Umbraco.Forms.Issues/issues/110)
+
+### Deploy 2.1.4 and 4.1.4 (September 7th 2021)
+
+*   Resolution of issue with failed extractions on vNext infrastructure.
+
+### [Deploy 4.2.0-rc001](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F4.2.0+) (August 19th 2021)
+
+*   Added support for deployment of form folders [#75 (Forms)](https://github.com/umbraco/Umbraco.Forms.Issues/issues/75)
+*   Added support for back-office transfer of data from custom packages or solutions
```
Comment (jonat123) [inline 10/umbraco-deploy/release-notes.md:240]:
```suggestion
*   Added support for backoffice transfer of data from custom packages or solutions 
```

**Comment by jonat123** [comment]
@AndyButland Hey! 

An update from Esha and me regarding this PR :) 

So we are going to merge in the PR, however, we have removed the release note pages from the summary file so it is not being shown on the docs page yet.

The reason for that is that we still want to do some changes to the way the release notes are structured before we release and make them public on the new platform.

Once we are done with the restructuring, we will let you know :-)

**Comment by AndyButland** [comment]
OK, sounds good - thanks.

**Pull request merged**